.github/workflows/auto-approve-dependabot-prs.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/github-script@v4
+    - uses: actions/github-script@v7
```

Comment:
This will need a change in line 17, because `pulls` is no longer directly accessible via `github` but only via `github.rest` (see [here](https://github.com/gitgitgadget/git/pull/1387/commits/1b933c241cf1e33faa171313aaef14f3f0e1c34f) for a thorough explanation).

Reply:
Done in https://github.com/gitgitgadget/gitgitgadget/compare/7f9ab3923da14e88556d9e5ac0eb9cb9b001bb0f..4c4f042bb991454a81b31342e7198526e7327fc2.